tutorial/detection/tag/tutorial-apriltag-detector.cpp

```diff
@@ -146,7 +146,7 @@ int main(int argc, const char **argv)
     if (std::string(argv[i]) == "--input" && i + 1 < argc) {
       opt_input_filename = std::string(argv[++i]);
     }
-    if (std::string(argv[i]) == "--tag-size" && i + 1 < argc) {
+    else if (std::string(argv[i]) == "--tag-size" && i + 1 < argc) {
```

Comment:
Without the `else`, it is not possible to run: `./tutorial-apriltag-detector --input AprilTag.jpg`